### PR TITLE
Upgrade node modules and Terraform

### PR DESCRIPTION
**Pull request by ericsorenson**

### CODE REVIEW

File: web-api/terraform/api/api-public.tf [hunk 1]
```diff
@@ -23,6 +23,8 @@ resource "aws_lambda_function" "api_public_lambda" {
 resource "aws_api_gateway_rest_api" "gateway_for_api_public" {
   name = "gateway_api_public_${var.environment}_${var.current_color}"
 
+  minimum_compression_size = "1"
+
```
Comment (mmarcotte) [inline web-api/terraform/api/api-public.tf:27]: Is this part of the gzip refactoring story?

Comment (ericsorenson) [reply web-api/terraform/api/api-public.tf:27]: Yes!

**Review by rachelschneiderman — APPROVED** [review]

File: web-api/terraform/api/api-public.tf (hunk 1 above)
Comment (mmarcotte) [reply web-api/terraform/api/api-public.tf:27]: cool, and it's harmless to include here? is there more work for that devex task?

Comment (ericsorenson) [reply web-api/terraform/api/api-public.tf:27]: It's harmless, I'm currently debugging to find the reason why the setting hasn't impacted the browser.

Comment (mmarcotte) [reply web-api/terraform/api/api-public.tf:27]: Thanks. Sorry that another PR snuck through and is now conflicting with your node modules update. 😢

**Review by kkoskelin — APPROVED** [review]
Our workflow seems to have introduced some of my accessibility-related changes into this PR (Tabs.jsx) but hopefully if we can merge this soon, there will not be merge conflicts with later efforts which also alter Tabs.jsx

**Pull request merged**